.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -30,16 +30,21 @@ on:
         description: Enable transaction broadcast
         type: boolean
         default: false
-
-      report_path:
```

Comment:
instead of removing, could be worth using [here](https://github.com/LedgerHQ/ledger-live/blob/develop/.github/workflows/test-ui-e2e-only-desktop.yml#L146)

Reply:
Since the input wasn't used before and because I reached maximum inputs for the workflow I removed it. But yes, it could be worth using it, going to look for a way to add it again :)